### PR TITLE
update gradle and give it more memory, latest corda gradle plugin

### PR DESCRIPTION
**Pull request by cedricwalter**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -113,8 +113,12 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
             "ch.cordalo:corda-common-contracts:$cordalo_version",
             "ch.cordalo:corda-common-workflows:$cordalo_version"
         ]
-
-
+        extraConfig = [
+                jvmArgs : [ "-Xmx1g"]
```
Comment (lolo8304) [inline build.gradle]: would add "-Xms1g"

File: gradle.properties [hunk 2]
```diff
@@ -1,3 +1,5 @@
 name=Test
 group=ch.cordalo.template
-version=0.1
\ No newline at end of file
+version=0.1
+
+org.gradle.jvmargs=-XX:+UseG1GC -Xmx1g -Dfile.encoding=UTF-8 -XX:+HeapDumpOnOutOfMemoryError
```
Comment (lolo8304) [inline gradle.properties]: would add -Xms1g

**Review by lolo8304 — CHANGES_REQUESTED** [review]
add -Xms

File: build.gradle (hunk 1 above)
Comment (cedricwalter) [reply build.gradle]: not supposed to be committed as not working

File: gradle.properties (hunk 2 above)
Comment (cedricwalter) [reply gradle.properties]: done

**Pull request merged**